### DISCUSSION TITLE
How do I turn on prefetch for bundle chunks?

### DISCUSSION THREAD

**Discussion by vr-santi** [vercel/next.js#39989] [Help]
In NextJS website, I see `prefetch` entries for bundle chunks:
![Screen Shot 2022-08-26 at 19 21 03](https://user-images.githubusercontent.com/101136729/186998064-436412cc-850d-4919-8e56-1ad23d41f0fe.png)

Not sure why I don't see them in my website. Also the docs don't say anything. Should I see them ? Am I doing something that might break it ? Thanks!

**Comment by gaston-flores** [comment]
Those tags are added by Next's [`Link`](https://nextjs.org/docs/api-reference/next/link) component (and [router.prefetch](https://nextjs.org/docs/api-reference/next/router#routerprefetch)), and only in production environments. Are you using `Link` to navigate in your app?
